conference/react-js/src/components/ForgotPassword/forgot-password-error-text.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import actions from '../../constants/actions-types';
+
+const ErrorText = ({ data }) => {
+  switch (data) {
```

Comment:
there is no need for switch operator as you have only two options. use if-else-if operator.
that will be fine to create some map. it will simplify file in general